model_lab_configs/huggingface/Intel/bert-base-uncased-mrpc/1/bert_trtrtx.json.config

```diff
@@ -0,0 +1,94 @@
+{
+    "oliveFile": "bert/bert_trtrtx_gpu.json",
+    "runtime": {
+        "autoGenerated": true,
+        "name": "Evaluate on",
+        "type": "enum",
+        "displayNames": [
+            "NVIDIA GPU (TRT RTX)",
+            "CPU"
+        ],
+        "path": "systems.local_system.accelerators.0.execution_providers.0",
+        "values": [
+            "NvTensorRTRTXExecutionProvider",
+            "CPUExecutionProvider"
```

Comment:
does it support cpu?

Reply:
I tried and it succeed. Does it make sense? Looks like a bit strange.
![image](https://github.com/user-attachments/assets/9ef74e51-0b2d-478e-930c-109709ecbe4a)